oshdb-util/src/main/java/org/heigit/ohsome/oshdb/util/time/OSHDBTimestampInterval.java

```diff
@@ -51,4 +52,9 @@ public boolean equals(Object o) {
         && this.fromTimestamp.equals(((OSHDBTimestampInterval) o).fromTimestamp)
         && this.toTimestamp.equals(((OSHDBTimestampInterval) o).toTimestamp);
```

Comment:
The equals and the compareTo don't fit to each other. pmd or sonarcloud will probably complain here as well. The problem is that in the compareTo the comparison is based only on the `fromTimestamp` and I would expect that it also includes the `toTimestamp`.
```java
var ti1 = new OSHDBTimeInterval(0,1);
var ti2 = new OSHDBTimeInterval(0,2);
ti1.compareTo(ti2) -> 0;
ti1.equals(t2) -> false;
```
This is nothing which needed to get fixed in this PR "code stye fixes" but we should to revisit this later.

Reply:
ok, can you maye create a ticket for it, so that we don't forget it?